### PR TITLE
feat: added doc name to doc dropdown selection

### LINKED ISSUE #360: remove the useless top button on the sidebar
Currently in doc for each part, there would be button linking to itself. For example for apisix, there would be an `Apache APISIX` on the top.
image

If users click it, it would direct them to the front page of the same section, so I think we don't need it. 
If we do need it, it might be a good idea to change its outlooking from others so people won't expect there is some kind of overview of apisix after clicking this button.

### PR DESCRIPTION
**Pull request by iamayushdas**
Fixes: #360 

Changes:

added the navigated doc page name to docs dropdown

Screenshots of the change:

scrnli_04_06_2021_22-40-22

live url: https://apisix-website-navigation.netlify.app/

### CODE REVIEW

**Comment by iamayushdas** [comment]
cc @Yiyiyimu @juzhiyuan

**Comment by Yiyiyimu** [comment]
@iamayushdas Cool! Thanks a lot!!! 🚀 

Found one problem is that `APISIX` and `dashboard` page still got their own links on the sidebar, others are good to me

**Comment by iamayushdas** [comment]
> @iamayushdas Cool! Thanks a lot!!! 🚀 
> 
> 
> 
> Found one problem is that `APISIX` and `dashboard` page still got their own links on the sidebar, others are good to me

Your wlcm, 
Yes that will be removed by there origin repo. Will do it

File: website/src/theme/DocPage/index.js [hunk 1]
```diff
@@ -26,6 +26,26 @@ function DocPageContent({ currentDocRoute, versionMetadata, children }) {
...
   } = versionMetadata;
+
+  useEffect(() => {
+    if(docsSidebars[sidebarName][0].label === "General"){
+      document.querySelectorAll(".navbar__link--active")[0].text = "General";
+    } else if (document.getElementById("getting-started")) {
+      document.querySelectorAll(".navbar__link--active")[0].text = "Apache APISIX";
+    } else if (document.getElementById("dashboard")) {
+      document.querySelectorAll(".navbar__link--active")[0].text = "Apache APISIX Dashboard";
+    } else if (document.getElementById("what-is-apisix-ingress-controller")) {
```
Comment (juzhiyuan) [inline website/src/theme/DocPage/index.js]: Maybe we could maintain a mapper here, like:

```
{
  "dashboard": "Apache APISIX Dashboard"
}
```

**Comment by juzhiyuan** [comment]
That's a good feature!!

File: website/src/theme/DocPage/index.js (hunk 1 above)
Comment (iamayushdas) [reply website/src/theme/DocPage/index.js]: Yes we can, 
Do you also want me to separate the selectors?

Comment (juzhiyuan) [reply website/src/theme/DocPage/index.js]: yes, make it more clearly

**Comment by Yiyiyimu** [comment]
Hi @iamayushdas could you update `apisix-website-navigation.netlify.app`

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**